generics/constants/common.js

```diff
@@ -66,5 +66,8 @@ module.exports = {
   "LINK" : "link",
   "DISCOVERED_BY_ME" : "discoveredByMe",
   "DEFAULT_SUBMISSION_REQUIRED" : 1,
-  "SCHOOL" : "school"
+  "SCHOOL" : "school",
+  "SUBENTITY" :"subEntity_",
+  "SUNBIRD_SERVER_TIMEOUT" : 5000,
+  "TIMEOUT_ERROR" : "Server timeout error"
```

Comment:
@VISHNUDAS-tunerlabs if its a response status/message add it in generics/https-status-codes